.claude/docs/CLAUDE.md_작성_가이드.md

```
@@ -0,0 +1,549 @@
+| **보편적 적용** | 모든 세션에 적용되는 정보만 포함 |
+| **Progressive Disclosure** | 세부 지침은 별도 파일로 분리 |
+| **자동 생성 금지** | `/init` 결과물은 시작점일 뿐, 반드시 수동 검토 |
+| **린터 역할 금지** | 코드 스타일은 린터에 맡기고, 훅으로 자동화 |
+
+---
+
+## 1️⃣ CLAUDE.md의 본질
+
+### LLM의 무상태성 이해
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)